qualification/conftest.py

```diff
@@ -44,7 +44,7 @@
 from .reporter import Reporter, custom_report_log
 
 logger = logging.getLogger(__name__)
-FULL_ANTENNAS = [1, 4, 8, 10, 16, 20, 32, 40, 55, 64, 65, 80]
+FULL_ANTENNAS = [4, 8, 20, 32, 40, 64, 80]
```

Comment:
Maybe let's leave 1 in: it can be a special case in some instances (no cross-correlations).